awswrangler/emr.py

```diff
@@ -130,6 +130,10 @@ def _build_cluster_args(**pars: Any) -> Dict[str, Any]:  # pylint: disable=too-m
         "StepConcurrencyLevel": pars["step_concurrency_level"],
     }
 
+    # Auto Termination Policy
+    if pars["auto_termination_policy"] is not None:
+        args["AutoTerminationPolicy"] = pars["auto_termination_policy"]
```

Comment:
Should we do a check here since we know that `pars["auto_termination_policy"]` should only have 1 key which is `IdleTimeout`?

Reply:
I'd probably let boto3 handle that validation / error - raising, but we can see what the rest of the team thinks.

Reply:
+1 boto3 will take care IMO